en/docs/reference/mediators/callout-mediator.md

```diff
@@ -201,7 +201,7 @@ The below example uses a Callout mediator to set a HTTP method when invoking a R
                <arg evaluator="xml" expression="$ctx:res"/>
             </args>
          </payloadFactory>
-         <send/>
+         <call/>
```

Comment:
Let's use respond mediator. Please test it and update.

Reply:
Modified the Xml script accordingly.